frontend/locales/fr.ts

```diff
@@ -1,7 +1,864 @@
+            },
+            step3: {
+                title: 'Paramètres additionnels',
+                continue: 'Continuer',
+                back: 'Retour',
+                optional: 'Optionnel',
+                links: 'Liens',
+                homepage: "Page d'accueil",
+                issues: 'Traqueur de problèmes',
+                source: 'Code source',
+                support: 'Support externe',
+                license: 'Licence',
+                type: 'Type',
+                customName: 'Nom',
+                url: 'URL',
+                seo: 'SEO',
+                keywords: 'Mots-clefs',
+            },
+            step4: {
+                title: 'Importer depuis Spigot',
+                continue: 'Continuer',
+                back: 'Retour',
+                optional: 'Optionnel',
+                convert: 'Convertir',
+                saveAsHomePage: "Sauvegarder comme page d'accueil",
+                convertLabels: {
+                    bbCode: 'Coller votre BBCode ici',
+                    output: 'Sortie Markdown',
+                },
+                preview: 'Aperçu',
+                tutorial: 'Comment obtenir le BBCode',
+                tutorialInstructions: {
+                    line1: 'Pour obtenir le BBCode de votre projet Spigot, suivez les instructions :',
+                    line2: '1. Allez sur votre projet et cliquez sur "Éditer la ressource".',
+                    line3: "2. Cliquez sur le symbole de la clef dans l'éditeur de description.",
+                    line4: '3. Copiez-collez le nouveau contenu dans la zone de texte supérieure du convertisseur, modifiez la sortie si vous le souhaitez et cliquez sur Enregistrer !',
+		},
+            },
+            step5: {
+                title: 'Finition',
+                text: 'Création...',
+            },
+            error: {
+                create: 'Il y a eu une erreur durant la création du projet',
+                nameExists: 'Un projet avec ce nom existe déjà',
+                slugExists: 'Un projet avec cette url réduite existe déjà',
+                invalidName: 'Ce nom contient des caractères invalides',
+                tooLongName: 'Le nom du projet est trop long',
+                tooLongDesc: 'La description du projet est trop longue',
+                tooManyKeywords: 'Le nom a trop de mots-clefs',
+                noCategory: 'Les projets doivent avoir une catégorie',
+                noDescription: 'Le project doit avoir une description',
+            },
+        },
+        sendForApproval: 'Envoyer pour approbation',
+        info: {
+            title: 'Information',
+            publishDate: 'Publier sur {0}',
+            views: '0 vues | {0} vue | {0} vues',
+            totalDownloads: '0 téléchargement au total | {0} téléchargement au total | {0} téléchargements au total',
+            stars: '0 étoiles | {0} étoile | {0} étoiles',
+            watchers: '0 observateurs | {0} observateur | {0} observateurs',
```

Comment:
```suggestion
            views: '0 vue | {0} vue | {0} vues',
            totalDownloads: '0 téléchargement au total | {0} téléchargement au total | {0} téléchargements au total',
            stars: '0 étoile | {0} étoile | {0} étoiles',
            watchers: '0 observateur | {0} observateur | {0} observateurs',
```
In English, the zero plural rule applies. In French, plural and singular usage are found, but normally, as soon as the number of elements is strictly less than 2, the singular should be used.